src/wait-for.js

```diff
@@ -25,6 +26,10 @@ function waitFor(
     showOriginalStackTrace = getConfig().showOriginalStackTrace,
     stackTraceError,
     interval = 50,
+    onTimeout = error => {
+      error.message = `${error.message}\n\n${prettyDOM(container)}`
```

Comment:
The reason we do this rather than simply calling `screen.debug()` here:

1. It could be that the `container` isn't bound to the same `document` as `screen`, so we should use the container given
2. If we do `screen.debug()` then the DOM output will appear above the error which I think would be confusing.